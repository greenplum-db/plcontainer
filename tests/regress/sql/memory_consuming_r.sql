CREATE OR REPLACE FUNCTION r_memory_test_c(num int) RETURNS void AS $$
# container: plc_r_shared
a <- matrix(1, ncol=1024, nrow=1024*num)
$$ LANGUAGE plcontainer;

select r_memory_test_c(10);
select r_memory_test_c(20);
select r_memory_test_c(40);
select r_memory_test_c(80);
select r_memory_test_c(100);
select r_memory_test_c(120);
