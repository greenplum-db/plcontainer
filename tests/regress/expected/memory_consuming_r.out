CREATE OR REPLACE FUNCTION r_memory_test_c(num int) RETURNS void AS $$
# container: plc_r_shared
a <- matrix(1, ncol=1024, nrow=1024*num)
$$ LANGUAGE plcontainer;
select r_memory_test_c(10);
 r_memory_test_c 
-----------------
 
(1 row)

select r_memory_test_c(20);
 r_memory_test_c 
-----------------
 
(1 row)

select r_memory_test_c(40);
 r_memory_test_c 
-----------------
 
(1 row)

select r_memory_test_c(80);
 r_memory_test_c 
-----------------
 
(1 row)

select r_memory_test_c(100);
 r_memory_test_c 
-----------------
 
(1 row)

select r_memory_test_c(120);
 r_memory_test_c 
-----------------
 
(1 row)

